salt/modules/linux_lvm.py

```diff
@@ -496,10 +496,14 @@ def lvcreate(
     if force:
         cmd.append("--yes")
 
-    out = __salt__["cmd.run"](cmd, python_shell=False).splitlines()
-    lvdev = "/dev/{0}/{1}".format(vgname, lvname)
+    out = __salt__['cmd.run_all'](cmd, python_shell=False)
```

Comment:
Please install a pre-commit hook as described in the docs https://docs.saltstack.com/en/latest/topics/development/contributing.html
This will automatically fix the formatting issues.

Reply:
Or check the CI `pre-commit` job output and update the code accordingly.

Reply:
Thanks I have installed this.

Reply:
Thanks I have installed this now.